### PR TITLE
feat(theme): retheme UI for Konflux

### PR DESCRIPTION
**Pull request by rohitkrai03**
## Fixes 

- https://issues.redhat.com/browse/HAC-5713

## Description

- Updates to the overall theme of the UI for Konflux.

## Type of change

- [x] Feature

## Screen shots / Gifs for design review 

https://github.com/openshift/hac-dev/assets/6041994/6f05ed4e-d98b-4846-98fe-12aca64d3aaa

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Review by sahil143 — CHANGES_REQUESTED** [review]
we should change the color for actions button also
image

====================================

remove this section from overview page
image

File: src/components/ApplicationDetails/ApplicationThumbnail.tsx [hunk 1]
```diff
@@ -11,7 +12,19 @@ import asset8 from '../../imgs/app-thumbnails/Asset 8.svg';
...
+  asset3,
+  asset4,
+  asset5,
+  asset6,
+  asset7,
+  asset8,
+  asset9,
+  asset10,
+  asset11,
+];
```
[the remark this answers is not in the document]
Comment (rohitkrai03) [reply src/components/ApplicationDetails/ApplicationThumbnail.tsx:27]: All the thumbnails are same. Just added one extra `asset11`. I've just replaced the SVG assets so everything should work fine with new or old applications.

**Comment by rohitkrai03** [comment]
> we should change the color for actions button also image
> 
> ====================================
> 
> remove this section from overview page image

Both of these are based on UX designs. There was no updates to the action buttons in UX designs.

**Comment by MariaLeonova** [comment]
Actions buttons should be treated in the same way as the primary button, great catch, @sahil143 !

And for the removal you are suggesting only the deployment card? I would agree, since we are removing everything that mentions deployments. I wonder if we could / should replace it with something else? possibly one of these: https://konflux-ci.dev/

**Comment by rohitkrai03** [comment]
Updated the action buttons - 
Screenshot 2024-05-15 at 7 15 36 PM

Used integration test content from konflux website.
Screenshot 2024-05-15 at 7 20 37 PM

@sahil143 @MariaLeonova

**Comment by abhinandan13jan** [comment]
This isn't related to this PR, but this quickstart is creating this issue. Changes looks good elsewhere
Screenshot 2024-05-15 at 8 40 22 PM

**Review by sahil143 — CHANGES_REQUESTED** [review]
On releases and User Access color of actions button is blue.
image

**Comment by rohitkrai03** [comment]
> On releases and User Access color of actions button is blue. image

Updated.

**Pull request merged**